src/Base/Support/Str.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Base\Support;
+
+class Str extends \Illuminate\Support\Str
+{
+    /**
+     * @param  string $value
+     * @param  int    $limit
+     * @param  string $end
+     *
+     * @return string
+     */
+    public static function limit($value, $limit = 100, $end = '...'): string
+    {
+        return parent::limit($value, $limit - strlen($end), $end);
```

Comment:
Jammer dat Illuminate er geen rekening mee houdt, maar zullen wij zelf nog iets ervoor zetten als
`if (strlen($value) <= $limit) return $value;`
, om onnodig afkappen te voorkomen in die enkele gevallen?